### ISSUE TITLE
dist/ManagerContext is breaking due to require

### ISSUE THREAD

**Issue by yoavniran** [davidtheclark/react-aria-menubutton#127]
hi,
im actually not a user of this lib but unfortunately am affected by what seems to be a bug in the transpiled code.

Im using [netlify-cms](https://github.com/netlify/netlify-cms) which is in turn using this library.

When it runs, it reaches this code:

node_modules/react-aria-menubutton/dist/ManagerContext.js

```javascript
var createReactContext = require('create-react-context');

var AriaMenuButtonManagerContext = createReactContext();
```

which breaks with the following error:

```
ManagerContext.js:5 Uncaught TypeError: createReactContext is not a function
    at Object../node_modules/react-aria-menubutton/dist/ManagerContext.js (ManagerContext.js:5)
    at __webpack_require__ (bootstrap:725)
    at fn (bootstrap:100)
    at Object../node_modules/react-aria-menubutton/dist/Wrapper.js (Wrapper.js:12)
    at __webpack_require__ (bootstrap:725)
    at fn (bootstrap:100)
    at Object../node_modules/react-aria-menubutton/dist/index.js (index.js:6)
    at __webpack_require__ (bootstrap:725)
    at fn (bootstrap:100)
    at Object../node_modules/netlify-cms-ui-default/dist/esm/Dropdown.js (Dropdown.js:22)

```

the code in dist/ManagerContext.js should be doing:

```javascript
var createReactContext = require('create-react-context').default;
```
which solves the problem.

Im not sure why/how this wasnt reported already here. I would expect this to be a problem for all users of this lib.

I would also recommend not using create-react-context anymore. it doesnt seem to be maintained.

**Comment by shirshendubhowmick** [comment]
@yoavniran Thanks for reporting this, will have a look. `create-react-context` was used to retain backward compatibility with older versions of react. We have an unreleased version which completely removes `create-react-context` and no longer backward compatible with older versions of React. We soon release that as a major version.

**Comment by nhardy** [comment]
@yoavniran I agree that
```javascript
var createReactContext = require('create-react-context').default;
```
should really be changed - but this surprised me since I'm running this library without issue.

I had a look and the version of `create-react-context` (0.3.0) depended on by `react-aria-menubutton` which seems to also add the ESM default export to `module.exports`. This means that it shouldn't matter if `.default` is appended or not.

Is something weird happening in `netlify-cms` that is requiring an older version of `create-react-context`?

**Comment by yoavniran** [comment]
thanks @nhardy 
Your comment made me take another look and I see now what's causing the issue (if not specifically why).

What I noticed is that create-react-context is being overriden by Gatsby's cache.
So its not the actual dependency (from node_modules/create-react-context) but a ES version that only exposes React's createContext method:

_.cache/create-react-context.js_

```javascript
import React from "react"

export default React.createContext
```

This explains why the require from react-aria-menubutton is breaking.

I dont know where this is coming from and if its possible to stop Gatsby from doing so. A google search I did didn't find anything useful. I will open an issue for them as well
